app/bundles/CampaignBundle/Config/services.php

```diff
@@ -23,5 +23,8 @@
     $services->load('Mautic\\CampaignBundle\\', '../')
         ->exclude('../{'.implode(',', array_merge(MauticCoreExtension::DEFAULT_EXCLUDES, $excludes)).'}');
 
+    $services->load('Mautic\\CampaignBundle\\Controller\\', '../Controller')
+        ->tag('controller.service_arguments');
```

Comment:
Isn't this done automatically by the Symfony Framework? If I comment it out and clear the cache then the campaign UI is working fine. Is there some edge case I'm missing?